src/aiida/cmdline/params/options/main.py

```diff
@@ -351,6 +352,17 @@ def set_log_level(_ctx, _param, value):
     help='In non-interactive mode, the CLI never prompts but simply uses default values for options that define one.',
 )
 
+INTERACTIVE = OverridableOption(
```

Comment:
Instead of adding a new option, why not update the existing `NON_INTERACTIVE` one to be a switch. You can update that to be `--interactive/--non-interactive` and set the default to be `--interactive` by doing `default=True`, to keep the current behavior. When adding it to the profile configure-rabbitmq command, you can do
```
@NON_INTERACTIVE(default=False)
```
to enforce non-interactive to be the default

that should work, shouldn't it? Would be better than having to maintain two options.